assets/javascripts/discourse/components/modal/assign-user.js

```diff
@@ -0,0 +1,37 @@
+import Component from "@glimmer/component";
+import { action } from "@ember/object";
+import { inject as service } from "@ember/service";
+import I18n from "I18n";
+
+export default class AssignUser extends Component {
+  @service taskActions;
+
+  formApi = {
```

Comment:
Given this is a little unusual, I think adding a comment would be good

```suggestion
  // `submit` property will be mutated by the `AssignUserForm` component
  formApi = {
```